docs/building-blocks/extending-templates/collector.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Implementing a property transformer and collector in an Azure resource manager template
+description: Describes how to implement a property transformer and collector in an Azure resource manager template
+author: petertay
+ms.date: 05/03/2017
+
+---
+
+# Implementing a property transformer and collector in an Azure resource manager template
```

Comment:
This title is confusing until after you read the doc ... Although I don't have a good suggestion for another title. Something like "Transform a template parameter into a resource schema" ? That isn't much better 😄 - I'm trying to think of a title that conveys the intent, more than the mechanism.